README.md

```diff
@@ -23,7 +23,8 @@ externally download sources for use by `npm` during `rpmbuild`.
 ## As OBS service
 
 - Get `package-lock.json` with `localfileVersion: 2`. For example,
-  - `npm install --package-lock-only --legacy-peer-deps` with npm 7+
+  - `npm install --package-lock-only --legacy-peer-deps --ignore-scripts`
+    with npm 7+
   - `--legacy-peer-deps` is required to fetch peer dependencies from remote
```

Comment:
Would it be possible to move this into another PR since it's unrelated here.

Reply:
Done!